src/mbgl/renderer/layers/render_line_layer.cpp

```diff
@@ -143,8 +143,8 @@ void RenderLineLayer::render(PaintParameters& parameters, RenderSource*) {
                      texsize,
                      crossfade,
                      parameters.pixelRatio),
-                     *posA,
-                     *posB,
+                     posA,
+                     posB,
```

Comment:
The `draw` lambda` accepts `optional`s anyway, so no need to dereference them on the way.